Sources/SolanaSwift/APIClient/APIRequest.swift

```diff
@@ -0,0 +1,65 @@
+import Foundation
+
+public protocol APIClientRequest: Encodable {
```

Comment:
Really, I have the same implementation)) But I use `Endpoint` instead of `Request` naming and it's more modern way nowadays

Reply:
@qwerkly maybe you are misunderstanding? this is the request model actually, it has no connection to Endpoint.